test/partialAttribute/nested-html/stamped_from_polymer_template.html

```diff
@@ -183,6 +184,27 @@
                     });
                 });
             });
+
+            describe('after `dom-bind` re-set the sub-partial property (just one scope) to the same instance but with new values', function () {
+                var scope, newVal = "still works";
+
+                beforeEach(function(done) {
+                    // IDEA: promisify juicy-html
+                    sinon.stub(importedTemplate, "_notifyPath");
+                    importedTemplate.addEventListener('stamping', function onceStamped(){
+                        importedTemplate.removeEventListener('stamping', onceStamped);
+                        scope = domBind.get('model.Page.scope1');
+                        scope.doesItWork = newVal;
```

Comment:
Is this line 197 needed?

Reply:
You are right, this line doesn't add anything meaningful to the test. I will remove it.